public/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
```

Comment:
In your wireframe, the button spans the entire width of the page above both the side-nav and the list of contact cards. The mark-up would look something like this:

```html
<div class="container">
  <div class="row">
    <div class="col s12">
      <!-- full width button -->
    </div>
    <div class="row">
      <div class="col s4">
        <!-- left nav -->
      </div>
      <div class="col s8">
        <div class="row">
          <div class="col s12">
            <div class="card">
              <!-- card HTML -->
            </div>
          </div>
        </div>
        <div class="row">
          <div class="col s12">
            <div class="card">
              <!-- card HTML -->
            </div>
          </div>
        </div>
        <div class="row">
          <div class="col s12">
            <div class="card">
              <!-- card HTML -->
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
```

Reply:
I've never used Materialize, BTW, so that might not be exactly the way Materialize wants things. But you can almost always nest grids like that.